ci/playbooks/molecule-prepare.yml

```diff
@@ -19,6 +19,7 @@
           - make
           - podman
           - python3
+          - python3-pip
```

Comment:
@cjeanner this issue is coming when python3-pip is not installed on RHEL-9.3 .
In our existing CI, https://review.rdoproject.org/zuul/build/415fd2fdd96143f39c49e58d83091e1e/console#7/0/6/controller
```
 pip install pip setuptools bindep --upgrade
```
pip cli is used in [setup_env](https://github.com/openstack-k8s-operators/ci-framework/blob/main/scripts/setup_env#L83) execution but as per the [code](https://github.com/openstack-k8s-operators/ci-framework/blob/main/scripts/setup_env#L42) it should use `PIP="${HOME}/test-python/bin/pip"`

that's why this issue is coming.
We are not seeing this issue in upstream as pip is already installed on the zuul node.

I think we need to set the value of pip here: https://github.com/openstack-k8s-operators/ci-framework/blob/main/scripts/setup_env#L75 to use venv pip to avoid this issue.

Reply:
ok, so the assumption pip was pre-installed was wrong then. Fair - approving.